mod/wikirate_source/set/type/source/source.rb

```diff
@@ -119,10 +135,14 @@ def parse_source_page url
   if errors.empty?
     preview = LinkThumbnailer.generate url
     if preview.images.length > 0
-     subcards["+image url" ] = preview.images.first.src.to_s
+      add_subcard '+image url', content: preview.images.first.src.to_s
+    end
+    unless subfield('title')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.